examples/voice_chat.c

```
@@ -1,246 +1,131 @@
-    
-    if (security_init(security_mode, CIPHER_AES_GCM) != 0) {
-        fprintf(stderr, "Failed to initialize security\n");
+
+    packet_init(OUR_NODE_ID);
+
+    /* Create a sample voice payload stub (in production this would be Codec2 frames) */
+    uint8_t voice_frame[128];
+    memset(voice_frame, 0x42, sizeof(voice_frame)); /* Dummy data */
+
+    /* Establish session with peer via handshake simulation */
+    uint8_t eph_pub[PUBLIC_KEY_LENGTH], eph_priv[PRIVATE_KEY_LENGTH];
```

Clang (14.0.6)

[warning] variable 'eph_pub' is not initialized

(cppcoreguidelines-init-variables)